e2e-tests/src/main.rs

```diff
@@ -125,15 +118,81 @@ fn test_token_transfer(config: Config) -> anyhow::Result<()> {
     Ok(())
 }
 
+// todo: check channeling tips
```

Comment:
This would be better as a task in Jira.

Reply:
surely, but I wanted to wait for the review, in case somebody come up with a better solution to be applied here

Reply:
Jira #329